test/parallel/test-exec-max-buffer.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
-require('../common');
-var exec = require('child_process').exec;
-var assert = require('assert');
 
-var cmd = 'echo "hello world"';
+const common = require('../common');
+const exec = require('child_process').exec;
+const assert = require('assert');
+
+const cmd = 'echo "hello world"';
 
 exec(cmd, { maxBuffer: 5 }, function(err, stdout, stderr) {
   assert.ok(err);
   assert.ok(/maxBuffer/.test(err.message));
 });
+
+if (!common.isWindows) {
+  const unicode = '中文测试'; // length: 12
+  exec('echo ' + unicode, {
+    encoding: 'utf8',
+    maxBuffer: 10
+  }, common.mustCall(function(err, stdout, stderr) {
+    assert(err);
+    assert.equal(err.message, 'stdout maxBuffer exceeded');
```

Comment:
Can you use `assert.strictEqual()` here. Also, you can drop the first assertion.